tests/unit/indexation/entity_formatter.js

```diff
@@ -0,0 +1,34 @@
+const CONFIG = require('config')
+const __ = CONFIG.universalPath
+const _ = __.require('builders', 'utils')
+const Q535 = require('./fixtures/Q535.json')
+require('should')
+const entityFormatter = __.require('db', 'elasticsearch/formatters/entity')
+
+describe('indexation: entity formatter', () => {
+  describe('flatten fields', () => {
+    it('should include words from inactive languages', async () => {
+      const doc = await entityFormatter(_.cloneDeep(Q535), { quick: true })
+      doc.flattenedLabels.includes('ᱦᱩᱜᱚ ᱵᱷᱤᱠᱴᱚᱨ').should.be.true()
+      doc.flattenedAliases.includes('Віктор-Мары').should.be.true()
+      doc.flattenedDescriptions.includes('թատերագիր').should.be.true()
```

Comment:
if we later open those languages in inventaire we would have to modify this test. I would suggest to only test if those `flattened*` are non empty strings

Reply:
that would strongly decrease the value of that test, to the point of not actually testing what the test label says. I would rather go with adding a fake language in the fixture, as we can safely assume that it will never be an active language: 18872b3b

Reply:
yeah it would decrease the relevance of this test, but i guess id rather have weaker tests than unstable ones.
this is non blocking